src/utils/vfs.js

```diff
@@ -176,10 +176,37 @@ const mountpointResolver = core => async (path) => {
  */
 const parseGet = req => {
   const {query} = url.parse(req.url, true);
-
-  return Promise.resolve({fields: query, files: {}});
+  const assembledQuery = assembleQueryData(query);
+  
+  return Promise.resolve({fields: assembledQuery, files: {}});
 };
 
+/*
+ * Assembles a given object query
+ */
+const assembleQueryData = (object) => {
```

Comment:
1. Place this function above `parseGet`.
2. Use `export const` here so that the function can be tested.

Reply:
Correction. I forgot this was not ESM. Don't add export, but place it in `module.exports` :D